### PR TITLE
feat(frontend): error boundaries

### PR DESCRIPTION
**Pull request by jeysal**

### CODE REVIEW

File: apps/frontend/src/App.tsx [hunk 1]
```diff
@@ -64,47 +65,66 @@ const ConfiguredLayout: React.FC = ({ children }) => {
...
-                      <Route path="/logout" component={Logout} />
+                <Route>
+                  <CheckAuth>
+                    <ConfiguredLayout>
+                      <ErrorBoundary>
+                        <React.Suspense fallback="Loading...">
+                          <Prefetch />
+                          <Switch>
+                            <Route exact path="/">
+                              <Home />
```
Comment (jeysal) [inline apps/frontend/src/App.tsx:97]: image

File: apps/frontend/src/__tests__/App.test.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import App from '../App';
+
+it('renders a loading indicator', async () => {
```
Comment (jeysal) [inline apps/frontend/src/__tests__/App.test.tsx:6]: It's better than nothing :smile:

**Comment by jeysal** [comment]
FFS this still has significant uncovered patch areas? o.O
Will have to see where tmrw

File: packages/react-components/src/molecules/ErrorCard.tsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        </Paragraph>
+        {error && (
+          <>
+            <Paragraph>
+              Should you ask for support via the menu, we would be glad if you
+              could paste the following info to help us track down the problem:
+            </Paragraph>
+            <figure css={{ margin: 0 }}>
+              <Caption>Error info for nerds</Caption>
+              <code css={errorInfoStyles}>
```
Comment (kmaid) [inline packages/react-components/src/molecules/ErrorCard.tsx:71]: 🤓

**Review by fampinheiro — APPROVED** [review]

**Pull request merged**